app/actions/app.js

```diff
@@ -1,6 +1,5 @@
 import { APP } from './actionsTypes';
 
-export const ROOT_OUTSIDE = 'outside';
```

Comment:
We should actually keep `outside` (not logged in) as an opposite to `inside` (logged in).
`NewServer` was created specifically to deal with adding a new server and hiding OnboardingView, so that's the one we should be removing.